corehq/apps/change_feed/management/commands/validate_kafka_pillow_checkpoints.py

```diff
@@ -30,7 +31,8 @@ def handle(self, **options):
 def validate_checkpoints(print_only):
 
     for pillow in get_all_pillow_instances():
-        if isinstance(pillow.get_change_feed(), KafkaChangeFeed):
+        if (pillow.pillow_id in getattr(settings, 'ACTIVE_PILLOW_NAMES', None) and
```

Comment:
I think it's better to replace ```None``` with ```[]``` because if ```ACTIVE_PILLOW_NAMES``` isn't defined in settings this statement will raise an error:
```TypeError: argument of type 'NoneType' is not iterable```

Reply:
Thanks, Updated